docs/running-node-ansible/genesis-vn.md

```diff
@@ -0,0 +1,75 @@
+# Running Genesis Validator Node using Ansible
+
+## Prerequisites
+Make sure you have all [prerequisites](./prerequisites.md) set up
+
+## Configure DCL network parameters (local machine)
+### 1. Set network chain id in [`deployment/ansible/inventory/hosts.yml`]
+```yaml
+all:
+  vars:
+    chain_id: <chain-id>
+  ...
+```
+Every network must have a unique chain ID (e.g. `test-net`, `main-net` etc.)
+
+## Configure node type specific parameters (local machine)
+### 1. Specify target instance address in the inventory file
```

Comment:
Maybe we can try using #### for all similar numbered lists? I think ### looks too huge...